### ISSUE TITLE
Screen curtain is not working on newer windows insider builds

### ISSUE THREAD

**Issue by akash07k** [nvaccess/nvda#12491]
### Steps to reproduce:
1. start NVDA
2. Press insert+printscreen or turn on the curtain by going to the settings
3. Observe
### Actual behavior:
When we turn on the screen curtain, nothing happens. although, NVDA reports like: "Temporary screen curtain, enabled untill next restart" but screen still stays as before and doesn't turn black.
### Expected behavior:
Screen curtain should work.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Version: alpha-22960,34e831bb
#### Windows version:
Windows 10 (Insider) build 21390
#### Name and version of other software in use when reproducing the issue:
NA
#### Other information about your system:
NA
#### Does the issue still occur after restarting your computer?
Yes, always
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, tried from 2020.4 to latest alpha, but no luck
#### If add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by akash07k** [comment]
@feerrenrut 
@leonardder 
@josephsl 
Guys, I'm sorry to ping you in this issue, but this is a bug which can impact many users and will be good if it gets fixed before 2021.1 release

**Comment by josephsl** [comment]
Hi, or rather, hope it gets resolved before Version 21H2 (Cobalt) ships to release preview Insiders provided that 21H2 is build 21000 or later. Thanks.

**Comment by leonardder** [comment]
I'm curious to know:

1. Whether the JAWS screen curtain suffers from this issue
2. Whether the windows magnifier works correctly
3. Whether you have wslg running. I got reports from @zersiax that starting a wsl gui application breaks the curtain.

**Comment by CyrilleB79** [comment]
> 2. Whether the windows magnifier works correctly

More specifically, does the color inversion feature work correctly? I think that it uses the same color transformation matrix API as the screen curtain.

**Comment by leonardder** [comment]
> > 1. Whether the windows magnifier works correctly
> 
> More specifically, does the color inversion feature work correctly? I think that it uses the same color transformation matrix API as the screen curtain.

True. Actually, we're using the same API and making the screen black this way. May be transforming to another color does still work.

**Comment by akash07k** [comment]
Please find comments inline
> I'm curious to know:
> 
> 1. Whether the JAWS screen curtain suffers from this issue
I don't have JAWS, if you say then I can install the demo and let you know.
> 2. Whether the windows magnifier works correctly
Not sure, I don't have any vision sadly
> 3. Whether you have wslg running. I got reports from @zersiax that starting a wsl gui application breaks the curtain.
No, never tried WSL gui on this instance of windows, not even WSL.

**Comment by seanbudd** [comment]
I've reproduced this using windows build 21390.1. The screen curtain causes the screen to freeze, i.e. no longer visually update, but is otherwise visible.

As this is an insiders issue, and likely an unintended bug that will be fixed by Microsoft before a stable Windows version is shipped, we think it's best to delay looking at this until 2021.2.

**Comment by seanbudd** [comment]
This is also confirmed to be an issue with the Windows 11 insider build (22000.65)

**Comment by zersiax** [comment]
I don't necessarily agree. While it is true that this is likely an unintended bug by Microsoft, I think it's still a good idea to at least track down what's causing the bug. Microsoft has a bit of a track record of taking quite a bit of time to fix unintended bugs at times, so if this is broken in insiders both for windows 10 and Windows 11, and we don't actually know what is causing this to happen, we can't report appropriate feedback either, which may very well mean this does not in fact get fixed on its own.
Another thing is that people who don't have light perception will have no idea this is happening. NVDA does  not throw an error, the sound plays as normal and the blind person running insider builds now does not know that their banking details may very well be visible for anyone to see while they assume their screen is off.

**Comment by josephsl** [comment]
Hi,

I'm going to rename this issue to point out that this is a Windows 11 issue.

Re: #12585 collection
